source/ways-of-working/common-pipeline/common-pipeline.html.md.erb

```diff
@@ -66,7 +66,7 @@ The environments shown in the diagram reflect CFT. Review the table below to see
 |-|-|
 |AAT|Staging|
 |Perftest|Test|
-|Preview|Dev|
+|Preview|Either Dev or Demo|
```

Comment:
dev reflects it, why the change here?
```suggestion
|Preview|Dev|
```

Reply:
Yeah great point, putting the "Either" here isn't ideal.

Well, what I thought was the dev environment I set up was actually called demo. Jenkins injected "demo" into the {env} variable, and the branch you need to name to do releases is called "demo"

I'd actually like to change this to "Demo" only, but I worried about backwards compatability if other people had something called Dev.

If we want to keep it as Dev I'm not fussed, but my experience was just a bit surprising.